Portal_Perfomance_Employes/Program.cs

```diff
@@ -1,5 +1,22 @@
+var DevelopmentSpecificOrigins = "_developmentSpecificOrigins";
+var ProductionSpecificOrigins = "_productionSpecificOrigins";
+
 var builder = WebApplication.CreateBuilder(args);
 
+builder.Services.AddCors(options =>
+{
+    options.AddPolicy(DevelopmentSpecificOrigins,
```

Comment:
Have you tested that it will work for both development and production? Test it somehow or tell how to test it